nodejs/eks/cluster.ts

```diff
@@ -192,12 +193,13 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
 
     // Add any requested StorageClasses.
     const storageClasses = args.storageClasses || {};
+    const userStorageClasses: UserStorageClasses = {};
     if (typeof storageClasses === "string") {
         const storageClass = { type: storageClasses, default: true };
-        createStorageClass(`${name.toLowerCase()}-${storageClasses}`, storageClass, { parent: parent, provider: provider });
+        userStorageClasses[`${storageClasses}`] = createStorageClass(`${name.toLowerCase()}-${storageClasses}`, storageClass, { parent: parent, provider: provider });
```

Comment:
This is equivalent to `[storageClasses]` - can we use that simpler form?

Reply:
Done